public/app/features/explore/QueryRowActions.tsx

```diff
@@ -35,17 +35,22 @@ export function QueryRowActions(props: Props) {
         </div>
       )}
       <div className="gf-form">
-        <button disabled={!canHide} className="gf-form-label gf-form-label--btn" onClick={onClickToggleHiddenQuery}>
+        <button
+          disabled={!canHide}
```

Comment:
Nitpick: I would try to clean up the naming here `canHide`, `hideQuery`, `onClickToggleHiddenQuery` or `Disable/enable query` there is variants here of the same thing and the code is starting to look inconsistent.
```suggestion
          disabled={!canDisable}
```